frontends/web/src/routes/new-settings/components/tabs.tsx

```diff
@@ -0,0 +1,88 @@
+import { ReactNode } from 'react';
+import { Main, Header } from '../../../components/layout';
+import { View, ViewContent } from '../../../components/view/view';
+import { MobileHeader } from './mobileheader';
+import i18n from '../../../i18n/i18n';
+import { NavLink } from 'react-router-dom';
+import styles from './tabs.module.css';
+import { route } from '../../../utils/route';
+import { SettingsItem } from './settingsItem/settingsItem';
+import { RightChevron } from '../../../components/icon';
+
+type TTabsProps = {
+  settingsTabsDetail: TTab[];
+}
+
+type TWithSettingsTabsProps = {
+  children: ReactNode
+}
+
+export type TTab = {
+  url: string;
+  name: string;
+}
+
+const settingsTabsDetail: TTab[] = [
+  { url: '/new-settings/appearance', name: i18n.t('settings.appearance') },
+  { url: '/new-settings/manage-accounts', name: 'Manage accounts' },
+  { url: '/new-settings/device-settings', name: 'Device settings' },
+  { url: '/new-settings/advanced-settings', name: 'Advanced settings' },
+  { url: '/new-settings/about', name: 'About' },
+];
+
+export const WithSettingsTabs = ({ children }: TWithSettingsTabsProps) => {
+  return (
+    <Main>
+      <div className="hide-on-small"><Header title={<h2>Settings</h2>} /></div>
+      <View fullscreen={false}>
+        <ViewContent>
+          <div className="show-on-small">
+            <MobileHeader settingsTabsDetail={settingsTabsDetail} />
+          </div>
+          <div className="hide-on-small">
+            <Tabs settingsTabsDetail={settingsTabsDetail} />
+          </div>
+          {children}
+        </ViewContent>
+      </View>
+    </Main>)
+  ;
+};
+
+/**
+ * The "index" page of the settings
+ * that will only be shown in mobile.
+ *
+ * The data will be the same as the "tabs"
+ * we see on Desktop.
+ **/
+
+export const MobileSettingsItems = () => {
+  return (
+    <>
+      {settingsTabsDetail.map(tab => (
+        <SettingsItem
+          settingName={tab.name}
+          onClick={() => route(tab.url)}
+          key={tab.url}
+          extraComponent={<RightChevron />}
+        />
+      ))}
+    </>
+  );
+};
```

Comment:
@thisconnect Here I didn't re-style the Tabs on mobile. Instead, I'll use `SettingsItem` for this. 

My explanation:
Re-styling tabs on mobile will actually cause redundancy because we already have this same exact component called `SettingsItem`. So there's no use / doesn't make sense to re-create that component again by re-styling tabs when we can already use the already available `SettingsItem`.

However, I used the `settingsTabsDetail` data which is also used for the tabs on desktop.  

**So this way, there's only 1 source of truth of the data both for tabs displayed on desktop, and the settings on mobile.**

IMO, this is the best way to achieve the mobile settings and the desktop tabs because:

1. We're using the already created component (SettingsItem)
2. Both (desktop) Tabs and MobileSettings use the same "source"  / "data" (`settingsTabsDetail`).

WDYT? 😄